src/cljs/infinitelives/pixi/animations.cljs

```diff
@@ -0,0 +1,103 @@
+                     :or {speed 2}}]
+  (go
+    (loop [f 0]
+      (let [pos (- 0 (Math.pow (Math/pow speed 1.5) (- 11 (/ f 11))))]
+        (s/set-x! sprite pos)
+        (<! (e/next-frame))
+        (when (< pos -1)
+          (recur (inc f)))))))
+
+(defn swipe-out [sprite & {:keys [speed]
+                     :or {speed 2}}]
+  (go
+    (let [[width _] (coord/get-window-size)
+          [_ _ text-width _] (s/get-rects sprite)
+          slide-width (+ width (* 2 text-width))]
+      (loop [f 0]
+        (let [pos (Math.pow speed (/ f 20))]
+          (s/set-x! sprite pos)
+          (<! (e/next-frame))
+          (when (< pos slide-width)
+            (recur (inc f))))))))
+
+(defn swipe [sprite & {:keys [speed pause loop? loop-while]
+                     :or {speed 2 pause 0 loop? false loop-while #(true)}}]
+  (go-while loop-while
+    (loop [_ 0]
+        (<! (swipe-in sprite :speed speed))
+        (<! (timeout (* pause 1000)))
+        (<! (swipe-out sprite :speed speed))
+      (when loop? (recur _)))))
+
+(defn push-in [sprite & {:keys [speed]
+                       :or {speed 2}}]
+  (go
+    (let [[ _ original-scale] (s/get-scale sprite)]
+      (loop [f 0]
+        (let [scale (Math.pow (/ f 10) speed)]
+          (s/set-scale! sprite scale)
+          (<! (e/next-frame))
+          (when (< scale original-scale)
+            (recur (inc f))))))))
+
+(defn push-over [sprite & {:keys [speed]
+                       :or {speed 2}}]
+  (go
+    (let [[ _ original-scale] (s/get-scale sprite)]
+      (loop [f 0]
+        (let [scale (+ original-scale (Math.pow (/ f 10) (Math.pow speed (/ f 15))))
+              alpha (/ (- (* speed 50) f) 100)]
+          (s/set-scale! sprite scale)
+          (s/set-alpha! sprite alpha)
+          (<! (e/next-frame))
+          (when (> alpha 0)
+            (recur (inc f))))))))
+
+(defn push-through [sprite & {:keys [speed pause]
+                       :or {speed 2 pause 0}}]
+  (go
+    (<! (push-in sprite :speed speed))
+    (<! (timeout (* pause 1000)))
+    (<! (push-over sprite :speed speed))))
+
+(defn scale-to [sprite & {:keys [speed to-scale]
+                    :or {speed 2 to-scale 3}}]
+  (go
+    (let [[ _ original-scale] (s/get-scale sprite)]
+      (loop [f 0]
+        (let [operator (if (< to-scale original-scale) - +)
+              comparitor (if (< to-scale original-scale) > <)
+              scale (operator original-scale (/ (* f speed) 20))]
+          (s/set-scale! sprite scale)
+          (<! (e/next-frame))
+          (when (comparitor scale to-scale)
+            (recur (inc f))))))))
+
+(defn bump [sprite & {:keys [speed increase-scale pause]
+                    :or {speed 3 increase-scale 1.5 pause 0}}]
+  (go
+    (let [[ _ original-scale] (s/get-scale sprite)]
+      (<! (scale-to sprite :speed speed :to-scale (* original-scale increase-scale)))
+      (<! (timeout (* pause 1000)))
+      (<! (scale-to sprite :speed speed :to-scale original-scale)))))
```

Comment:
Agreed. 
I thought they might be handy for game jams for drop in text animations but they are likely too specific for a general library and probably work best as copy paste drop-in's.